pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -48,16 +48,14 @@ func Open(DBPath string) (db *DB, err error) {
 		return nil, err
 	}
 
+	// TODO add "PRAGMA journal_mode = WAL" somehow
 	sqlite, err := sql.Open("sqlite3", fmt.Sprintf("file:%s", DBPath))
```

Comment:
https://github.com/mattn/go-sqlite3#connection-string, maybe `?_journal=wal`

Reply:
that might work